### ISSUE TITLE
Canada.ca theme: Signon and wb-cont

### ISSUE THREAD

**Issue by juliemercier** [wet-boew/GCWeb#1147]
On the signon page `https://wet-boew.github.io/themes-dist/GCWeb/content-signedon-en.html`, can we have the **Skip to main content** link go to the signon section rather than the `<h1>` tag? A screen reader user would be unaware that they have successfully signed on and that they have options to modify their setting. I realize that they can  find this information using other keyboard functions but including this as the first item in the main content would be the polite thing to do?

**Comment by hsrudnicki** [comment]
Does this remain an issue or otherwise?

**Comment by rlambert27** [comment]
Yes this is still an issue :-)

**Comment by grifare** [comment]
Perhaps that was not the case 3 years ago but the h1 there now is very clear and reads "Content page - Signed On". 
I may be wrong but I cannot see what a screen reader user might misunderstand. Is this still an issue to be kept open?
